docs/source/release_notes.rst

```diff
@@ -32,6 +32,10 @@ v0.5.1 Jul 22, 2021
     Thanks to the following people for contributing to this release:
     :user:`davesque`, :user:`frances-h`, :user:`jeff-hernandez`, :user:`simha104`, :user:`tamargrey`, :user:`thehomebrewnerd`
 
+Breaking Changes
+++++++++++++++++
+    * The criteria for categorical type inference have changed (:pr:`1065`)
```

Comment:
Usually for breaking changes we put the one-liner with the PR number in the Changes section above and in Breaking Changes provide more details about how the change works / will impact users

Reply:
https://github.com/alteryx/woodwork/pull/1065/commits/0b346dad2e24e6ea39f27aa8db187a20b5f63574